python/dgl/backend/numpy/tensor.py

```diff
@@ -1,23 +1,25 @@
 from __future__ import absolute_import
 
-import numpy as np
+import numpy as onp
+from mxnet import numpy as np
+import mxnet as mx
```

Comment:
What are differences between mxnet.numpy backend and mxnet backend?

Reply:
`mxnet.numpy` is a numpy-like interface for MXNet 2.0: https://numpy.mxnet.io/, the goal of this project is to make MXNet compatible with numpy and accelerate basic operations with TVM. It's still under development.